agenta-backend/agenta_backend/models/api/organization_models.py

```diff
@@ -19,9 +18,17 @@ class Organization(BaseModel):
     invitations: Optional[List]
 
 
+class CreateOrganization(BaseModel):
```

Comment:
how is it working right now without this ?

Reply:
@aakrem previoulsy, there was no feature to create an Organization. We simply created it at the time of signup.